content/pages/meetup/2023/0913-nycu.rst

```diff
@@ -0,0 +1,93 @@
+========================================
+Meetup 2023 September 6th in NYCU
```

Comment:
@jamie212 , the date in the subject is incorrect.

Reply:
> @jamie212 , the date in the subject is incorrect.

Thank you for pointing it out, I've corrected it.